### PR TITLE
Staging tests

### PR DESCRIPTION
**Pull request by ns212**

### CODE REVIEW

File: .env.staging [hunk 1]
```diff
@@ -17,9 +17,9 @@ export BITCOIN_CORE_NETWORK="testnet";
 # TODO: Set the following values
```
Comment (nud3l) [inline .env.staging:17]: If the values below are OK, please remove the TODO note

**Comment by nud3l** [comment]
@ns212 @savudani8 should we rerun the tests here and merge or how do you want to proceed?

**Comment by savudani8** [comment]
@ns212  Not sure it's actually `source`-ing the env file, it's logging this: `2022-01-24 14:38:18          API-WS: disconnected from ws://127.0.0.1:9944: 1006:: connection failed`

**Review by savudani8 — CHANGES_REQUESTED** [review]
Maybe `set -a` before sourcing the env file?

**Pull request closed without merging**